### DISCUSSION TITLE
Custom Tailwind styles in v3?

### DISCUSSION THREAD

**Discussion by emmadesilva** [filamentphp/filament#8451] [Help] [answered]
So, when creating custom Blade pages I obviously need some custom Tailwind styles, but I'm not finding any information on the v3 docs on how to set up Vite to compile the frontend styles. Am I crazy, or do I need to create a custom theme or something?

Not finding any issues or discussions on the matter either, so asking here hoping to get some clarity. Cheers!

**Comment by leandrocfe** [comment, marked as answer]
Hi, take a look
https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme

**Reply by emmadesilva** [reply]
Okay, seems like this is what I need to do. Thanks!
